client/js/screens/Profile/styles.js

```diff
@@ -0,0 +1,42 @@
+import theme from "../../config/globalStyles";
+
+const styles = {
+  container: {
+    ...theme.container,
+    height: "100%",
+    justifyContent: "flex-start"
+  },
+  title: {
+    fontSize: theme.subheadingFontSize,
+    color: theme.black,
+    fontWeight: "bold"
+  },
+  link: {
+    ...theme.link
+  },
+  profilePic: {
+    width: 150,
+    height: 150,
+    borderRadius: 75
```

Comment:
Please use flex with `resizeMode: "contain"` instead of fixed height and width.

Reply:
Issue solved.